src/extensions/default/AutoUpdate/UpdateInfoBar.js

```diff
@@ -113,6 +117,29 @@ define(function (require, exports, module) {
                 }
             }
         }
+        // Content Container Width between Icon Container and Button Container or Close Icon Container
+        // will be assigned when window will be rezied.
+        var resizeContentContainer = function () {
+            if($updateContent.length > 0 && $contentContainer.length > 0 && $updateBar.length > 0) {
+                var newWidth = $updateBar.outerWidth() - 38;
+                if($buttonContainer.length > 0) {
+                    newWidth = newWidth- $buttonContainer.outerWidth();
+                }
+                if($iconContainer.length > 0) {
+                    newWidth = newWidth - $iconContainer.outerWidth();
+                }
+                if($closeIconContainer.length > 0) {
+                    newWidth = newWidth - $closeIconContainer.outerWidth();
+                }
+
+                $contentContainer.css({
+                    "maxWidth": newWidth
+                });
+            }
+        };
+
+        resizeContentContainer();
```

Comment:
Suggestion: Resizing is an expensive operation, I think you can use  _.debounce function here for performance sake. Especially useful when a lot of processing is happening on the main thread.

https://davidwalsh.name/javascript-debounce-function#comment-80304